settings/tui/screen/requests.go

```diff
@@ -0,0 +1,44 @@
+package screen
+
+import (
+	"github.com/GDGVIT/socrates/schema"
+	"log"
+	"encoding/json"
+	"net/http"
+	"bytes"
+)
+
+func (s *Screen) putUpdate() {
+	client := &http.Client{}
+
+	body, err := json.Marshal(s.config)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	req, err := http.NewRequest(http.MethodPut, "http://localhost:" + s.port + "/update", bytes.NewBuffer(body))
```

Comment:
Have there be a global config for feeding in host ans port in information. It'll add the ability to decouple the backends even more

Reply:
There is a global config being loaded in `main`. I added a port parameter to `Screen` for better readability, and passed this parameter to the constructor call in `main`.

Reply:
Adding a host parameter to the viper config will ensure decoupling if someone wants it

Reply:
Host parameter? What would that denote?